lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
+  }
+
+  .nx-card-header {
+    @include container-vertical;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  .nx-card__call-out {
+    margin-bottom: $nx-spacing-xs;
+  }
+
+  .nx-card__text {
+    margin-bottom: $nx-spacing-l;
+  }
+
+  .nx-card-footer {
+    border-top: 1px solid #d1d6f0;
+    padding-top: $nx-spacing-l;
+    width: 100%;
+  }
+}
+
+// Horizontally orientated cards appear in a column
+.nx-card-container--column {
+  @include container-vertical;
+
+  flex-direction: column;
+
+  .nx-card {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: auto 1fr auto;
+    grid-template-columns: auto 1fr auto;
+    -ms-grid-rows: auto auto;
+    grid-template-areas:
+      "card-callout card-title card-footer"
+      "card-callout card-text card-footer";
+
+    margin: $nx-spacing-md 0;
+    text-align: left;
+    width: 100%;
+  }
+
+  .nx-card-header {
+    -ms-grid-row: 1;
+    -ms-grid-column: 2;
+    grid-area: card-title;
+
+    @include container-vertical;
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl $nx-spacing-md $nx-spacing-xxl;
+  }
+
+  .nx-card__text {
+    -ms-grid-row: 2;
+    -ms-grid-column: 2;
+    grid-area: card-text;
+
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl;
+  }
+
+  .nx-card__call-out {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: card-callout;
+    -ms-grid-column-align: center;
+    justify-self: center;
+
+    margin: 0;
+    padding: 0 $nx-spacing-xxl 0 0;
+  }
+
+  .nx-card-footer {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 3;
+    grid-area: card-footer;
+    -ms-grid-column-align: start;
+    justify-self: start;
+
+    border: none;
+    margin: 0;
+    padding: 0 0 0 $nx-spacing-xxl;
+    // necessary to make the link wrap
+    max-width: 150px;
+  }
+}
+
+// Styles for the alternate vertical style
+.nx-card-container--row {
+  .nx-card--vertical-alt {
+    .nx-card-content {
+      align-items: center;
+      display: flex;
+      -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+      column-gap: 24px;
+    }
+
+    .nx-card__call-out {
+      @include container-horizontal;
+      display: inline-block;
+      flex-grow: 1;
+      margin-bottom: $nx-spacing-md;
+      text-align: right;
+    }
+
+    .nx-card__text {
+      display: inline-block;
+      flex-grow: 1;
+      margin-bottom: $nx-spacing-md;
+      text-align: left;
```

Comment:
This poses a problem when it automatically switches to normal vertical layout.  At that point I assume the text should be center aligned.  I doubt there's a way to distinguish that in CSS though.

Reply:
> This poses a problem when it automatically switches to normal vertical layout. At that point I assume the text should be center aligned. I doubt there's a way to distinguish that in CSS though.

As discussed in Slack I removed the wrapping from the alternate examples. https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/f691bf1fa5d8590b2a0336cbca00eca16e23f909